src/lib/containers/CardContainer.tsx

```diff
@@ -102,15 +103,7 @@ export const CardContainer = (props: CardContainerProps) => {
     }
     // else show "no results" UI (see PORTALS-1497)
     return (
-      <>
-        <p className="SRC-no-results-title">
-          There is currently no content here.
-        </p>
-        <p className="SRC-no-results-description">
-          Information is always being updated, so check back later to see if
-          content has been added.
-        </p>
-      </>
+      <NoContentAvailable />
```

Comment:
Refactored into a reusable component, then decided against reusing it (for the case when no people are found in the UserCardListRotate component).  So, it's available to you, but unused here.